construct/lib/container.py

```diff
@@ -28,8 +28,11 @@ class Container(dict):
     """
     __slots__ = ["__keys_order__"]
 
-    def __init__(self, **kw):
+    def __init__(self, *args, **kw):
```

Comment:
I'll have to get to that later. I don't see any test for Container, currently, and it's kind of a beast.